docs/guides/deployment.rst

```diff
@@ -68,3 +68,139 @@ deployment set at `hoodie-app-tracker/deployment.md`_.
 .. _Hoodie Tracker App: https://github.com/hoodiehq/hoodie-app-tracker
 .. _hoodie-app-tracker/deployment.md: https://github.com/hoodiehq/hoodie-app-tracker/blob/master/deployment.md
 
+
+DEPLOYMENT IN LINUX
+===================
+
+This guide is for Linux only at this point.
+I have tried to deploy [Hoodie-App-Tracker](https://github.com/hoodiehq/hoodie-app-tracker) as an example:
+
+INSTALL DEPENDENCIES
+~~~~~~~~~~~~~~~~~~~~~
+
+1. `Install CouchDB`_ 1.2.0 or later, 1.4.0 or later recommended for performance.
+
+2. `Install NodeJS`_ 0.10.0 or later.
+
+3. `Install nginx`_, any recent version will do.
+
+4. `Install Monit`_, any recent version will do.
+
+5. `Install git`_.
+
+.. _Install CouchDB: http://linoxide.com/linux-how-to/install-couchdb-futon-ubuntu-1604/
+
+.. _Install NodeJS: https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-16-04
+
+.. _Install nginx: https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-16-04
+
+.. _Install Monit: https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-monit
+
+.. _Install git: https://www.digitalocean.com/community/tutorials/how-to-install-git-on-ubuntu-16-04
+
+COUCHDB
+~~~~~~~
+
+We assume you set up CouchDB with your package manager or manually following the
+`installation procedure`_.
+
+In order to test if CouchDB is running fine or not, we can simply run the following
+command which will retrieve the information through curl.
+::
+
+$   curl localhost:5984
+
+If you are already using CouchDB for other things, we recommend starting a second
+instance of CouchDB that is completely separate from your original one. See below
+for instructions.
+
+In this guide, we assume that your CouchDB is available at `port 5984`_.
+
+Create a CouchDB admin user called **admin** with a strong password of your choice at
+by clicking on the *Fix this* at `Apache CouchDB-Futon:Overview`_ link in the
+lower right corner. Keep that password in mind.
+
+Next we have to change CouchDB’s default configuration on a few points. The easiest thing is to go to and change the following fields (double click a value to enter the editing mode):
+
+   couchdb -> delayed_commits: false
+   couchdb -> max_dbs_open: 1024
+   couch_httpd_auth -> timeout: 1209600 ; that’s two weeks
```

Comment:
we don’t need the `couch_httpd_auth` setting any more, can you remove it please?